config/config.js

```diff
@@ -0,0 +1,30 @@
+import env from "../config/env.generated.json";
+
+let config = {
```

Comment:
I'd probably rename this since it's not really setting up a `config`, it technically exports an API library, right? Maybe `userSettings.js`